### DISCUSSION TITLE
Can't get dark mode going?

### DISCUSSION THREAD

**Discussion by nmoore1** [jlesage/docker-qdirstat#10] [Q&A] [answered]
Hi, could anyone let me know what dark mode setting should look like in compase or .env? I can't seem to get it working. Thanks.

**Comment by jlesage** [comment, marked as answer]
There are 2 ways to set environment variables in Docker compose:
```
environment:
  DARK_MODE: 1
```
or
```
environment:
  - DARK_MODE=1
```
See https://docs.docker.com/compose/compose-file/compose-file-v3/#environment for more details.
